x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/scalar/datetime/QuarterProcessor.java

```diff
@@ -21,15 +21,15 @@
     public static final String NAME = "q";
     private static final DateTimeFormatter QUARTER_FORMAT = DateTimeFormatter.ofPattern("q", Locale.ROOT);
 
-    
+
```

Comment:
Formatting whitespace is just noise... only the lines touched need to appear in a commit.

Reply:
This was committed by mistake, sorry.
I had some changes which I reverted and I ended up with this noise here.